composer.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "bldr-io/php-block",
+    "description": "Block for handling PHP-related tasks",
+    "authors": [
+        {
+            "name": "Ameen Ross",
+            "email": "a.ross@amdev.eu"
+        }
+    ],
+    "minimum-stability": "dev",
+    "require": {
+        "jakub-onderka/php-parallel-lint": "0.*"
+    },
+    "require-dev": {
+        "bldr-io/bldr": "~7.0",
```

Comment:
Just updated to 8.0.0 the other day. as a heads up. This should still work on 7.0 though